### ISSUE TITLE
Accessibility of CSS styling on documentation pages

### ISSUE THREAD

**Issue by imrek** [jasmine/jasmine#1122]
Please consider **changing the background color** of the [code samples](http://jasmine.github.io/2.4/introduction.html) in the documentation, they are hard to read, as the the background and some of the foreground colors lack enough contrast. I can still read it, but it's just requires way more concentration, and I can well imagine that people with visual impairment will inevitably have problems with it.

![screenshot from 2016-05-23 08-07-07](https://cloud.githubusercontent.com/assets/13435019/15461735/8895bdae-20bd-11e6-91b7-12460e4d0369.png)

**Comment by slackersoft** [comment]
The source for the [Jasmine documentation](https://jasmine.github.io) has its [own github repository](https://github.com/jasmine/jasmine.github.io). We do accept [pull requests](https://github.com/jasmine/jasmine.github.io/pulls) for our documentation, if you'd like to help out, or [issues](https://github.com/jasmine/jasmine.github.io/issues) if you have things you'd like see addressed but aren't sure how.

**Comment by sgravrock** [comment]
We've changed the text and background colors so that everything has at least a 4.5:1 contrast ratio, which should solve the problem. If you still think the contrast is too low, please let us know. Thanks for using Jasmine!
